samples/snippets/src/main/java/aiplatform/CreateFeatureSample.java

```diff
@@ -0,0 +1,90 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Create a single feature for an existing entity type. See
+ * https://cloud.google.com/vertex-ai/docs/featurestore/setup before running 
+ * the code snippet
+ */
+
+package aiplatform;
+
+// [START aiplatform_create_feature_sample]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.aiplatform.v1.CreateFeatureOperationMetadata;
+import com.google.cloud.aiplatform.v1.CreateFeatureRequest;
+import com.google.cloud.aiplatform.v1.EntityTypeName;
+import com.google.cloud.aiplatform.v1.Feature;
+import com.google.cloud.aiplatform.v1.Feature.ValueType;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceClient;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceSettings;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateFeatureSample {
+
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featurestoreId = "YOUR_FEATURESTORE_ID";
+    String entityTypeId = "YOUR_ENTITY_TYPE_ID";
+    String featureId = "YOUR_FEATURE_ID";
+    String description = "YOUR_FEATURE_DESCRIPTION";
+    String valueType = "YOUR_FEATURE_VALUE_TYPE";
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+    int timeout = 300;
+    createFeatureSample(project, featurestoreId, entityTypeId, featureId, description, valueType,
+        location, endpoint, timeout);
+  }
+
+  static void createFeatureSample(String project, String featurestoreId, String entityTypeId,
+      String featureId, String description, String valueType, String location, String endpoint,
+      int timeout) throws IOException, InterruptedException, ExecutionException, TimeoutException {
+
+    FeaturestoreServiceSettings featurestoreServiceSettings =
+        FeaturestoreServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (FeaturestoreServiceClient featurestoreServiceClient =
+        FeaturestoreServiceClient.create(featurestoreServiceSettings)) {
+
+      Feature feature = Feature.newBuilder().setDescription(description)
+          .setValueType(ValueType.valueOf(valueType))
```

Comment:
nit: can we use the ENUM directly. Applies to all occurrences.

Reply:
Sure, will pass the ENUM variable instead of String Variable as below
```suggestion
          ValueType valueType = ValueType.STRING;
          
          ...
          .setValueType(valueType)
```